contrib/spec/podman.spec.in

```diff
@@ -200,6 +200,11 @@ BuildArch: noarch
 BuildRequires: python3-devel
 BuildRequires: python3-setuptools
 BuildRequires: python3-varlink
+
+Requires: python3-setuptools
```

Comment:
Do you need setuptools to run?  It would be nice if this was just a build-time dependency.

Reply:
agree

Reply:
I use pkg_resources to pull information from the egg at runtime.